### PR TITLE
decompile rsrc files in the repository

### PR DESCRIPTION
**Pull request by jmairboeck**
They are now compiled during the build.

### CODE REVIEW

File: FindLib/etc/libfind.rdef
Comment (jmairboeck) [inline FindLib/etc/libfind.rdef:1]: This one doesn't seem to be used.

Comment (korli) [reply FindLib/etc/libfind.rdef:1]: It doesn't hurt to keep it?

Comment (jmairboeck) [reply FindLib/etc/libfind.rdef:1]: That's why I still included it. I just wanted to remark that this isn't used.

Comment (korli) [reply FindLib/etc/libfind.rdef:1]: I understand, just wanted to be sure.

**Pull request merged**